presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisClientManager.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kinesis;
+
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import com.amazonaws.services.kinesis.AmazonKinesisClient;
+import com.amazonaws.services.kinesis.model.DescribeStreamRequest;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.google.inject.Inject;
+import io.airlift.log.Logger;
+
+/**
+ * Creates and manages AWS clients for this connector.
+ * <p>
+ * Note: credentials can be supplied explicitly through the configuration.  However when these are
+ * omitted, the default AWS provider chain is used (which includes instance profile credentials).
+ */
+public class KinesisClientManager
+        implements KinesisClientProvider
+{
+    private static final Logger log = Logger.get(KinesisClientManager.class);
+    private final AmazonKinesisClient client;
+    private final KinesisAwsCredentials kinesisAwsCredentials;
+    private final AmazonS3Client amazonS3Client;
+    private final AmazonDynamoDBClient dynamoDBClient;              // for Checkpointing
+
+    @Inject
+    KinesisClientManager(KinesisConfig kinesisConfig)
+    {
+        log.info("Creating new client for Consumer");
```

Comment:
This log statement doesn't add much value. I'd remove it.

Reply:
removed